### DISCUSSION TITLE
controller:greenpower: Error: Parameter 'tempMaster' is missing

### DISCUSSION THREAD

**Discussion by Gildas224** [Koenkk/zigbee2mqtt#23274] [Devices]
Hello,
I have bought the power tag R9M60, but I can't add it to Zigbee2MQTT because I have permanently the fault : "controller:greenpower: Error: Parameter 'tempMaster' is missing". What I have to do to add ithis material to my devices of Zigbee2MQTT?
Thank you

**Comment by Koenkk** [comment]
Let's continue in https://github.com/Koenkk/zigbee2mqtt/issues/21306
